docs/sources/guides/whats-new-in-v6-7.md

```diff
@@ -74,13 +80,17 @@ rendering_timeout = 10s
 concurrent_render_limit = 10
 ```
 
-### Export dashboard as pdf
-This feature allows you to export the dashboard as a PDF document. All dashboard panels will be rendered as images and added into the PDF document.
+### Export dashboard as PDF
+
```

Comment:
What documentation was updated for this feature? Please link.

Reply:
@alexanderzobnin is working on it, please link here when done

Reply:
Pushed docs in #22746, so should be like 
```markdown
Learn more in [Export dashboard as PDF]({{< relref "../enterprise/export-pdf/" >}})
```